authentication/tests/test_user_login.py

```diff
@@ -0,0 +1,62 @@
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from authentication.models import CustomUser
+
+
+class UserLoginAPITests(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.test_user = CustomUser.objects.create_user(
+            person_email="test@test.com",
+            password="Test1234",
+            person_name="Test",
+            person_surname="Test",
+        )
+        cls.test_user.is_active = True
+        cls.test_user.save()
+
+    def test_login_successful(self):
+        url = "/api/auth/token/login/"
+        data = {
+            "person_email": "test@test.com",
+            "password": "Test1234",
+        }
+        response = self.client.post(url, data)
+        self.assertEqual(response.status_code,
+                         status.HTTP_200_OK)
+        self.assertContains(response, "auth_token")
```

Comment:
I recommend to have at least one entry in unittests, containing a full JSON returned by BE.

Thus, unittests can be used as a references (for example — a kind of documentation for FE team, or for QC).

In case if there is stuff content of which is hard to predict, you case use `AnyStr`, `AnyInt` etc classes (see example project).

Reply:
I've used approach as in example project and added one more assert with full JSON.
Thanks.